packages/catalog-model/src/kinds/GroupEntityV1alpha1.ts

```diff
@@ -18,6 +18,17 @@ import type { Entity } from '../entity/Entity';
 import schema from '../schema/kinds/Group.v1alpha1.schema.json';
 import { ajvCompiledJsonSchemaValidator } from './util';
 
+/**
+ * Backstage Group Profile.
+ *
+ * @public
+ */
+export type GroupProfile = Record<string, string> & {
```

Comment:
This addition of opening up the profile with any additional keys and at the same time encoding that the 3 existing ones are special is quite scary. It's likely to lock us into this shape forever unless we ship breaking changes. I'd like to explore other options here where we keep the profile information as it is today. Some related discussion over here as well: #5930

I think a big part of the issue here is that while it's possible to easily customize entity schema, it's not as easy to customize the rendering of things in the frontend. In many places we're simply relying on complete replacements of components right now, but that might change with #11819

Reply:
That should be updated now